.github/workflows/gradle.yml

```diff
@@ -12,15 +12,22 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
 
     steps:
     - uses: actions/checkout@v2
     - name: Set up JDK 1.8
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
+    - name: Set up XCode
+      uses: maxim-lobanov/setup-xcode@master
+      with:
+        xcode-version: '12.2'
```

Comment:
current xcode version is `12.4`. 12.4 on the ci should work because it worked on my machine.

Reply:
Changed in https://github.com/CUTR-at-USF/opentripplanner-client-library/pull/12/commits/cece4885e2f7085b31b425af51e091d0df87470a